server/README.md

```diff
@@ -0,0 +1,74 @@
+# Eclipse BaSyx Python SDK - HTTP Server
+
+This repository contains a Dockerfile to spin up an exemplary HTTP/REST server following the [Specification of the AAS Part 2 API][6] with ease.
```

Comment:
> This repository

"repository" is now the wrong word. Maybe "module" or "package"? We should decide for one name for our subdirectories and stick with it consistently.

Reply:
I would chose package, since a single python file is already a module.